iquip/apps/scheduler.py

```diff
@@ -302,7 +299,8 @@ def __init__(self, name: str, parent: Optional[QObject] = None):
         """Extended."""
         super().__init__(name, parent=parent)
         self.schedulerFrame = SchedulerFrame()
-        self.schedulerFrame.queueView.fetched_event.connect(self.displayMenu)
+        self.schedulerFrame.queueView.rightButtonPressed.connect(self.displayMenu)
+        self.addExperiment(ExperimentInfo("HI", {"rid": 1, "priority": 1}))
```

Comment:
Is this for testing?

Reply:
Oh, I forgot to delete it after testing the code 😅